advanced_functionality/mxnet_mnist_byom/mxnet_mnist.ipynb

```diff
@@ -131,9 +129,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "from mnist import train\n",
```

Comment:
Can we change to:
```
model = train(data = data, num_cpus=0, num_gpus=1)
```
And add a note in the markdown above that says something like, "If running this in a non-GPU notebook instance, please adjust num_gpus=0 and num_cpu=1?